--- src/Makefile.in	2006-04-17 07:53:07.000000000 +0200
+++ src/Makefile.in-new	2007-04-14 18:48:59.507798170 +0200
@@ -11,7 +11,8 @@
 
 CC = @CC@
 CFLAGS = @CFLAGS@
-LDFLAGS = @LDFLAGS@ @LIBS@ -lutil -lid3
+LDFLAGS = @LDFLAGS@
+LDADD = @LIBS@ -lutil -lid3
 INSTALL = @INSTALL@
 
 CFILES = main.c\
@@ -36,7 +37,7 @@
 all: ripperX
 
 ripperX: $(OFILES)
-	$(CC) $(LDFLAGS) $(OFILES) -o ripperX
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OFILES) -o ripperX $(LDADD)
 
 .c.o:
 	$(CC) $(CFLAGS) -c $<
